src/app/shared/case-editor/case-edit-submit.html

```diff
@@ -55,6 +55,31 @@ <h2 class="plain-message">Check the information below carefully. This will form
       </tbody>
     </table>
   </ng-container>
+  <ng-container *ngIf="readOnlySummaryfieldsToDisplayExists()">
+    
+    <table class="form-table">
+      <tbody>
+        <ng-container *ngFor="let field of this.sortedFields">          
+            <tr *ngIf="!(field | ccdIsCompound); else ReadOnlyCompoundRow" ccdLabelSubstitutor [caseField]="field"
```

Comment:
can we use NgSwitch instead of this construct? that's the recommended angular approach for this scenarios and I find it much more easy to understand. There's an example in case-viewer.component